test-apps/display-performance-test-app/src/backend/DisplayPerfRpcImpl.ts

```diff
@@ -179,7 +179,7 @@ export default class DisplayPerfRpcImpl extends DisplayPerfRpcInterface {
       DisplayPerfRpcInterface.backendServer.close();
 
     if (DisplayPerfRpcInterface.chrome)
-      await DisplayPerfRpcInterface.chrome.kill();
+      DisplayPerfRpcInterface.chrome.kill();
```

Comment:
Motivation? `chrome.kill` is async.

Reply:
chrome-laucnher was updated and so were its types
![image](https://github.com/iTwin/itwinjs-core/assets/11051042/7d96a659-672d-41a7-8947-0919d277a955)

<img width="1127" alt="image" src="https://github.com/iTwin/itwinjs-core/assets/11051042/7d3b5c7d-509f-4eb6-8e37-0056e246b031">

Reply:
Ok. They didn't update [their documentation](https://github.com/GoogleChrome/chrome-launcher#launchthenchrome--).